### PR TITLE
Update extensions.md

### PR DESCRIPTION
**Pull request by DaveAtKitronik**
Added in Kitronik packages as all were missing.

### CODE REVIEW

File: docs/extensions.md [hunk 1]
```diff
@@ -67,8 +87,18 @@ Check out [microbit.org](http://microbit.org/resellers/) for more information on
...
+},
+{
+  "name": "Kitronik :GAME Controller",
+  "url":"https://github.com/KitronikLtd/pxt-kitronik-game-controller",
+  "cardType": "package"
+},
+{
   "name": "Sparkfun Gamer:bit",
-  "url":"/pkg/sparkfun/pxt-gamer-bit",
+  "url":"/KitronikLtd/pxt-kitronik-zip-64",
```
Comment (pelikhan) [inline docs/extensions.md]: Intentional?

Comment (DaveAtKitronik) [reply docs/extensions.md]: oops nope. 
actually line 91 points our picture at sparkfuns url…

Comment (DaveAtKitronik) [reply docs/extensions.md]: switched them round to have correct URL with correct product.

File: docs/extensions.md [hunk 2]
```diff
@@ -88,7 +88,7 @@ Check out [microbit.org](http://microbit.org/resellers/) for more information on
 ```codecard
 [{
   "name": "Kitronik :GAME ZIP64",
-  "url":"http:s//pkg/sparkfun/pxt-gamer-bit",
+  "url":"https://github.com/KitronikLtd/pxt-kitronik-zip-64",
```
Comment (pelikhan) [inline docs/extensions.md]: Use pkg url format

Comment (DaveAtKitronik) [reply docs/extensions.md]: I tried to use a pkg url but I couldn't make it work for our packages. 
If you can advise the correct one then I'll change them.

Comment (pelikhan) [reply docs/extensions.md]: Why URL did you try? If your package is allowed and you have a README.md file, it should render it properly.

Comment (DaveAtKitronik) [reply docs/extensions.md]: I tried the url "pkg/kitronikltd/pxt-kitronik-zip-64" but maybe I tried it in the wrong place - in the editor add a package. Will investigate some more

Comment (pelikhan) [reply docs/extensions.md]: make sure to have the leading / ....
```
"url": "/pkg/kitronikltd/pxt-kitronik-zip-64"
```

**Pull request merged**